src/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+file(GLOB srcsrc "*.f90") # add all *.f90 files
+#list(REMOVE_ITEM ${srcsrc} "${CMAKE_CURRENT_SOURCE_DIR}/quick_api_test.f90" )
+
+foreach(src ${srcsrc})
```

Comment:
I think that you should be able to combine this logic (and the code in modules/ and subs/) into a single file(GLOB_RECURSE) expression (plus maybe some exclusions and/or REMOVE_ITEMs on the list).

Reply:
I fixed it by `file(GLOB srcsrc ${CMAKE_CURRENT_SOURCE_DIR}/*.f90 ${CMAKE_CURRENT_SOURCE_DIR}/subs/*.f90 ${CMAKE_CURRENT_SOURCE_DIR}/modules/*.f90)`